scripts/setup-gcp.sh

```
@@ -0,0 +1,71 @@
+
+# Script to set up GCP Artifact Registry for Cloud Run deployment
+
+# Check if gcloud is installed
+if ! command -v gcloud &> /dev/null; then
+    echo "Error: gcloud CLI is not installed. Please install it first."
+    echo "Visit: https://cloud.google.com/sdk/docs/install"
+    exit 1
+fi
+
+# Prompt for GCP project ID
+read -p "Enter your GCP Project ID: " PROJECT_ID
```

Shellcheck (0.10.0)

[info] read without -r will mangle backslashes.

(SC2162)